### DISCUSSION TITLE
EcoCode Rules for C#

### DISCUSSION THREAD

**Discussion by StuAalto** [green-code-initiative/ecoCode#277] [General]
Hello everyone.
First of all, thank you for the work you've done on the existing plugins.

Are you planning to develop a plugin for C#?

Regards,

Bertrand

**Comment by dedece35** [comment]
Hi @StuAalto,

yes a new plugin C# is currently starting.
do you want to contribute to it ? 

regards,
David

**Comment by StuAalto** [comment]
Hi @dedece35 .

Thank you for your reply, which is excellent news for the team concerned.
I'm just the messenger and not at all a C# specialist. 

However, I will pass on the request to the team interested in this particular subject. 
It would be a great way to get them even more involved and contribute to the project (killing two birds with one stone).

I'll keep you informed.

regards,

Bertrand

**Reply by olegoaer** [reply]
@StuAalto Take a look at [our new plugin C#](https://github.com/green-code-initiative/ecoCode-csharp). Enjoy.
